pyproject.toml

```diff
@@ -12,7 +12,7 @@ python = "~3.8"
 prometheus-client = "0.7.1"
 requests = "2.31.0"
 confluent-kafka = "1.5.0"
-insights-core = "3.2.3"
+insights-core = "3.2.9"
```

Comment:
Since we have a new update to the `OSRelease` combiner this week and it will be included in the new `insights-core v3.2.10` tag. Will it be possible to update the version to `3.2.10` here?

Reply:
Sure I will update it.